### ISSUE TITLE
你好  测试如何进行

### ISSUE THREAD

**Issue by neverstoplearn** [jiaxiaojunQAQ/LAS-AT#4]
通过README命令：
`python3 LAS_AT_train_cifar10.py --model WideResNet --epsilon_types 3 4 5 6 7 8 9 10 11 12 13 14 15 --attack_iters_types 3 4 5 6 7 8 9 10 11 12 13 14 --step_size_types 1 2 3 4 5 --epochs 110 --data-dir cifar-data --out-dir CIFAR10/LAS_PGD_AT`
进行训练了LAS_PGD_AT模型，进行测试时结果只有0.1，测试提供的模型结果正常，训练的日志记录结果也正常，可能是什么原因，谢谢。

**Comment by neverstoplearn** [comment]
测试命令：python3 test_CIFAR10.py --model_path model.pth --out_dir ./output/ --data-dir cifar-data
测试结果：
Files already downloaded and verified
Files already downloaded and verified
0.03137254901960784
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
AT_models_test_acc: 0.1
AT_fgsm_acc: 0.1
AT_pgd_acc_10: 0.0993
AT_pgd_acc_20: 0.0988
AT_pgd_acc_50: 0.0982
AT_pgd_cw_acc_20: 0.0962

**Comment by neverstoplearn** [comment]
提供的模型测试结果：
Files already downloaded and verified
Files already downloaded and verified
0.03137254901960784
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
tensor([[[0.0314]],

        [[0.0314]],

        [[0.0314]]], device='cuda:0')
AT_models_test_acc: 0.8623
AT_fgsm_acc: 0.6898
AT_pgd_acc_10: 0.5762
AT_pgd_acc_20: 0.5648
AT_pgd_acc_50: 0.5615
AT_pgd_cw_acc_20: 0.5572

**Comment by neverstoplearn** [comment]
最后部分的训练日志：
[2024/05/29 17:02:24] - Epoch    Seconds         LR              Train Loss      Train Acc
[2024/05/29 17:26:05] - 107      1420.5                  0.0010          1.6196          0.7278
[2024/05/29 17:29:14] - Test Loss        Test Acc        PGD Loss        PGD Acc
[2024/05/29 17:29:14] - 0.7389           0.8615          1.0344          0.5653
[2024/05/29 17:29:14] - Test Acc         PGD Acc
[2024/05/29 17:29:14] - 0.8610           0.5705
[2024/05/29 17:29:14] - Epoch    Seconds         LR              Train Loss      Train Acc
[2024/05/29 17:53:02] - 108      1428.2                  0.0010          1.6174          0.7296
[2024/05/29 17:56:11] - Test Loss        Test Acc        PGD Loss        PGD Acc
[2024/05/29 17:56:11] - 0.7401           0.8616          1.0359          0.5660
[2024/05/29 17:56:11] - Test Acc         PGD Acc
[2024/05/29 17:56:11] - 0.8610           0.5705
[2024/05/29 17:56:11] - Epoch    Seconds         LR              Train Loss      Train Acc
[2024/05/29 18:19:51] - 109      1420.3                  0.0010          1.6146          0.7315
[2024/05/29 18:23:00] - Test Loss        Test Acc        PGD Loss        PGD Acc
[2024/05/29 18:23:00] - 0.7411           0.8624          1.0370          0.5640
[2024/05/29 18:23:00] - Test Acc         PGD Acc
[2024/05/29 18:23:00] - 0.8610           0.5705
[2024/05/29 18:23:00] - [0.429, 0.4686, 0.4878, 0.4705, 0.4762, 0.4847, 0.4606, 0.4771, 0.4831, 0.4787, 0.4827, 0.48, 0.4709, 0.4779, 0.4864, 0.4693, 0.4756, 0.4886, 0.4652, 0.474, 0.4783, 0.4411, 0.4663, 0.4778, 0.4774, 0.4775, 0.4886, 0.4868, 0.4701, 0.4604, 0.4673, 0.481, 0.4837, 0.468, 0.4558, 0.488, 0.4464, 0.4546, 0.4825, 0.4444, 0.491, 0.5636, 0.5642, 0.5697, 0.5679, 0.5651, 0.5692, 0.5705, 0.5691, 0.5698, 0.5666, 0.5642, 0.5657, 0.5653, 0.566, 0.564]
[2024/05/29 18:23:00] - [0.7417, 0.7615, 0.7675, 0.7737, 0.7557, 0.7811, 0.7466, 0.7684, 0.7771, 0.7584, 0.7805, 0.7571, 0.769, 0.7786, 0.7652, 0.7675, 0.7725, 0.7804, 0.7598, 0.7809, 0.7715, 0.7449, 0.7657, 0.7828, 0.7654, 0.7831, 0.7858, 0.7611, 0.7744, 0.7637, 0.7823, 0.7724, 0.7803, 0.752, 0.7637, 0.7801, 0.7788, 0.7857, 0.7817, 0.7598, 0.7931, 0.8448, 0.8501, 0.8495, 0.8561, 0.8539, 0.8598, 0.861, 0.8593, 0.8594, 0.8618, 0.8625, 0.8609, 0.8615, 0.8616, 0.8624]

**Comment by jiaxiaojunQAQ** [comment]
你需要check一下模型的参数是否正常取出，有些是保存在state_dict里面

**Comment by neverstoplearn** [comment]
> 你需要check一下模型的参数是否正常取出，有些是保存在state_dict里面

已解决 感谢
